features/section_explicit_waiting.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Waiting for a Section
+
+  I want to be able to explicitly wait for a section
+  In order to interact with the element once it is ready
+
+  Background:
+    When I navigate to the home page
```

Comment:
You should have a ```Given``` statement somewhere first, before using ```When```

Reply:
These navigation steps are kind of `Given` and kind of `When`. We should probably standardise them at some point.

If you've got some time after this going in, that can be a quick PR you can do?